magefiles/magefile.go

```diff
@@ -356,7 +356,11 @@ func (Docs) Serve() error {
 
 // Generate generates CLI references
 func (Docs) Generate() error {
-	// TODO
+	
+	if err := sh.Run("go", "run", "docs/doc_creator/main.go"); err != nil {
+		  
+		return err
+	}
```

Comment:
```suggestion
	err := doc.GenMarkdownTree(commands.NewApp("v0.38"), "./docs/docs/references/cli")
	if err != nil {
		return err
	}
```

And the version is fixed to v0.38. We should take it from git tag.